### PR TITLE
NXT-221 Fix issue with functional tests not running

### PR DESCRIPTION
**Pull request by johndavey72**
https://nicedigital.atlassian.net/browse/NXT-221

### CODE REVIEW

File: functional-tests/wdio.conf.ts [hunk 1]
```diff
@@ -41,10 +43,12 @@ export const config: WebdriverIO.Config = {
 	cucumberOpts: {
 		require: [
 			"./steps/**/*.ts",
-			"./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",
+			"./steps/given.ts",
+			"./support/pagePaths.ts",
+			// "./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",
```
Comment (mgallagher1980) [inline functional-tests/wdio.conf.ts:48]: Should we remove this completley? No need for it anymore now we have removed wdio-cucumber-steps

File: functional-tests/wdio.conf.ts [hunk 2]
```diff
@@ -41,10 +43,12 @@ export const config: WebdriverIO.Config = {
 	cucumberOpts: {
 		require: [
 			"./steps/**/*.ts",
-			"./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",
+			"./steps/given.ts",
```
Comment (mgallagher1980) [inline functional-tests/wdio.conf.ts:46]: Do we need this with the catch all above?

File: web/src/pages/forms/__snapshots__/life-sciences-contact-us.page.test.tsx.snap
Comment (mgallagher1980) [inline web/src/pages/forms/__snapshots__/life-sciences-contact-us.page.test.tsx.snap:1]: Is this part of this PR?

File: web/src/pages/forms/life-sciences-contact-us.page.test.tsx
Comment (mgallagher1980) [inline web/src/pages/forms/life-sciences-contact-us.page.test.tsx:1]: Is this part of this PR?

File: web/src/pages/forms/life-sciences-contact-us.page.tsx
Comment (mgallagher1980) [inline web/src/pages/forms/life-sciences-contact-us.page.tsx:1]: Is this part of this PR?

**Review by mgallagher1980 — CHANGES_REQUESTED** [review]
Also, could we add in a When.ts file to the steps folder as it would be good to have one for the future

File: functional-tests/wdio.conf.ts (hunk 1 above)
Comment (mgallagher1980) [reply functional-tests/wdio.conf.ts:48]: I am referring to line 48 - // "./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",

File: functional-tests/wdio.conf.ts (hunk 2 above)
Comment (mgallagher1980) [reply functional-tests/wdio.conf.ts:46]: If we do, could we specify the other Then and When files

File: functional-tests/wdio.conf.ts (hunk 1 above)
Comment (johndavey72) [reply functional-tests/wdio.conf.ts:48]: yes will clean that up Martin

File: web/src/pages/forms/__snapshots__/life-sciences-contact-us.page.test.tsx.snap
Comment (johndavey72) [reply web/src/pages/forms/__snapshots__/life-sciences-contact-us.page.test.tsx.snap:1]: hmmm shouldn't be in there, good spot.

File: web/src/pages/forms/life-sciences-contact-us.page.tsx
Comment (johndavey72) [reply web/src/pages/forms/life-sciences-contact-us.page.tsx:1]: Good spot again!

**Comment by johndavey72** [comment]
> Also, could we add in a When.ts file to the steps folder as it would be good to have one for the future

There's one in the codebase already on the main branch, it's not in the PR as I haven't made any changes to it.

**Review by lizaadebowale — APPROVED** [review]

**Review by kavithanice — APPROVED** [review]

**Review by lizaadebowale — APPROVED** [review]

**Pull request closed without merging**